elements/urth-core-behaviors/logging-behavior.html

```diff
@@ -0,0 +1,103 @@
+<!--
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+-->
+
+<!--
+This behavior is used to encapsulate Notebook specific properties
+-->
+<script>
+    'use strict';
+    (function() {
+        var Urth = window.Urth = window.Urth || {};
+
+        /**
+         * Behavior that ability to control the output of logs. It redefines the logging methods in Polymer.Base to
+         * control their invocation based on the log levels set in the `log` property.
+         *
+         * @group Urth Core
+         * @polymerBehavior Urth.LoggingBehavior
+         */
+        Urth.LoggingBehavior = {
+
+            properties: {
+
+                /**
+                 * Comma separated string of log levels to allow. The levels are INFO, WARN, DEBUG, ERROR, ALL.
+                 * An empty string or unset property is interpreted no logs
+                 */
+                log: {
+                    type: String,
+                    value: "",
+                    reflectToAttribute: true
+                },
+
+                _logBooleanMap: {
+                    type: Object,
+                    computed: '_toLogBooleanMap(log)'
+                }
+            },
+
+            _toLogBooleanMap: function(log){
+
+                var _logBooleanMap = {};
+                (log || "").split(',')
+                    .map(function(part){return part.trim().toUpperCase();})
+                    .filter(function(part){
+                        return !!part;
+                    })
+                    .forEach(function(level){
+                        _logBooleanMap[level] = true;
+                    });
+
+                return _logBooleanMap;
+
+            },
+
+            _log: function(){
+              this._info.apply(this, arguments);
+            },
+
+            _info: function() {
```

Comment:
My only hesitation is with using an underscore prefix for what is actually the public API for this behavior. I'm assuming underscore is used because some of the API names could be common in other elements and something like `logInfo` or `infoLog` is verbose/confusing.

At a minimum should provide some minimal documentation for these API, even though they'll show up as Private.